app/utils/config.js

```diff
@@ -158,10 +158,37 @@ export default EmberObject.create({
   LANGUAGE_DE: {
     uri: 'http://publications.europa.eu/resource/authority/language/DEU',
   },
-  ACTIVITY_TYPE_URIS: {
-    vertalen: 'http://kanselarij.vo.data.gift/id/concept/activiteit-types/e9a532a5-65c1-484f-9288-1720dcd1296c',
-    handtekenen: 'http://kanselarij.vo.data.gift/id/concept/activiteit-types/6a43f78e-c835-40a2-bf32-81209ac6e42a',
-    drukproeven: 'http://kanselarij.vo.data.gift/id/concept/activiteit-types/165a56c5-6d32-4a8e-aefe-a1755eb0c0a4',
+  ACTIVITY_TYPES: {
```

Comment:
Hier staan andere dingen dan [hier](https://github.com/kanselarij-vlaanderen/kaleidos-project/blob/e026e4f6738c02ab592ffe46925974f9fcae59c2/config/resources/dossier-domain.lisp#L90). Dat moeten we uitklaren. Weet jij meer @ValenberghsSven ?

Reply:
Hier linken we naar deze:
https://github.com/kanselarij-vlaanderen/kaleidos-project/pull/134/files